File: help/pt_pt/exercicios_da_ficha_2/Ficha02_004/Ficha02_004.c
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <stdio.h>
#include <string.h>
#include <math.h>
//Main Function
int main() {
    printf("Qual o seu nome \t: ");
    char nome[256];
    fgets(nome,256, stdin);
    printf("Qual a sua idade \t: ");
    int idade;
    scanf("%d", &idade);
    printf("Olá %s  de %d anos",nome ,idade );
    return 0;
}


//C Language c11  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 set 07 16:09

File: help/pt_pt/exercicios_da_ficha_2/Ficha02_004/Ficha02_004.java
// Programador          Ant0ni0 M@ns0 - Ant0ni0 M@ns0

import java.util.Scanner;

public class Ficha02_004 {

    static Scanner keyb = new Scanner(System.in);//keyboard

    //Main Function
    public static void main( String[] args) {
        System.out.print("Qual o seu nome \t: " );
        String nome = keyb.nextLine();
        System.out.print("Qual a sua idade \t: " );
        int idade = Integer.valueOf(keyb.nextLine());
        System.out.print("Olá " + nome + " de " + idade + " anos" );
    }

}
//Java  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 set 07 16:09
